concreep-refilled/control.lua

```diff
@@ -159,8 +165,8 @@ function _reinit_reentrant_level1_body()
 	debug_print_coroutine_was_called("_reinit_reentrant_level1_body()")
 
 	-- pseudo: for global.reinit_surface_index, #global.reinit_surfaces do
-		-- -- limit how many surfaces checked per call
-		-- for pass_index = 1, SURFACES_PER_REINIT_PASS do
+		-- -- limit how many settings.global["allowed surfaces"].value checked per call
+		-- for pass_index = 1, DEFAULT_SURFACES_PER_REINIT_PASS do

```

Comment:
should reference the correct settings.global here

Reply:
I definitely mixed something up with the replace all option today.. sigh thanks for pointing out! :)

Reply:
done